cmd/influx_tools/internal/format/binary/common.go

```diff
@@ -1,6 +1,7 @@
 package binary
 
-//go:generate sh -c "protoc -I$(go list -f '{{ .Dir }}' -m github.com/gogo/protobuf) -I. --gogofaster_out=Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types:. binary.proto"
+// TODO(dstrand1): re-enable codegen for binary.proto when influx_tools is ported to new protobuf lib
+// go:generate sh -c "protoc -I$(go list -f '{{ .Dir }}' -m github.com/gogo/protobuf) -I. --gogofaster_out=Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types:. binary.proto"
```

Comment:
Is there a good reason to disable this rather than leaving in place until we can fix it up?

Reply:
Originally I disabled it because of issues with the protoc binary and the static code checks job, I can try to re-enable it and see what happens